docs/content/codeformat.fsx

```diff
@@ -24,7 +24,7 @@ can be called to format snippets repeatedly:
 *)
 
 let fsharpCompiler = Assembly.Load("FSharp.Compiler")
-let formattingAgent = CodeFormat.CreateAgent(fsharpCompiler)
+let formattingAgent = CodeFormat.CreateAgent()
```

Comment:
Out of curiosity, is this a needed change for some reason? (Happy to delete the parameter if it's not needed, but then we should probably delete the whole line 26.)